.c8rc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "reporter": ["lcovonly", "cobertura"],
+    "reporter": ["lcovonly", "cobertura", "v8"],

```

Comment:
I don't know how slow it is to always generate this; if it's expensive maybe we should just have a flag to customize the reporters we ask for through c8.